contents/docs/product-analytics/troubleshooting.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: Troubleshooting and FAQs
+---
+
+Having trouble with events? There are a few common reasons that you may not see events appear in your project.
+
+### 1. The library is being used incorrectly. (this is usually the issue)
+
+> Warning: Make sure to not use sensitive data while debugging using third-party tools.
+
+1. Visit [webhook.site](https://webhook.site/) and copy “Your unique URL”.
+2. Inside your code replace [posthog.com](https://app.posthog.com) (or [eu.posthog.com](https://eu.posthog.com) if using the EU Cloud) with “Your unique URL”.
+3. Make sure that the SDK sends a couple of events.
+4. See if the incoming events arrive as expected.
+
+![Events visible on webhook.site](../../images/docs/product-analytics/events-in-webhook.site.png)
+
+### 2. There is a bug within the library being used.
+
+#### Front-end libraries
+1. Open the devtools and switch to the network tab.
+2. Look for the request to app.posthog.com/e (or eu.posthog.com/e if using the EU Cloud) that appears to be incorrect.
+
+![network request being sent to posthog](../../images/docs/product-analytics/network-request-to-posthog.png)
+
+3. See how to [Report your specific issue](/docs/product-analytics/troubleshooting#report-your-specific-issue) to get further support.
+
+#### Backend libraries
+See the respective debugging guides, e.g. [Node.js debugging](https://posthog.com/docs/libraries/node#debugging-and-exceptions).
+
+### 3. Events are not ingested after being received by PostHog.
+
+#### Front-end libraries
+1. Open the devtools and switch to the network tab.
+2. Right click one of the network requests to posthog.com/e (or eu.posthog.com/e if using the EU Cloud) and select Copy -> Copy as cURL.
+3. Format the request as specified in [Report your specific issue](/docs/product-analytics/troubleshooting#report-your-specific-issue)
+4. Try to send the cURL directly.
+5. Wait 1 hour and check if the event is still not visible in PostHog. Also make sure PostHog cloud is not experiencing an [active incident](https://status.posthog.com) as that could cause an ingestion lag.
+6. See how to [Report your specific issue](/docs/product-analytics/troubleshooting#report-your-specific-issue) to get further support.
+
+#### Backend libraries
+1. Track your outgoing network requests using something like [Wireshark](https://www.wireshark.org/).
+2. Follow the 3. step from the Front-end section.
+
+## Report your specific issue
+
```

Comment:
Again, it would be nice to have a short paragraph describing what we'll be doing in the steps below and the goal of it. 

e.g., If none of the above fixes solved your problem, you can report your issue. Be sure to include the cURL command etc..